brave/brave-core
# Storybook: Created reusable-panel-component

Resolves <https://github.com/brave/brave-browser/issues/14932>

Created a reusable panel and added to storybook

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. `npm run storybook`
2. Confirm that the storybook shows new component.
3. Check functionality of the new component.

https://user-images.githubusercontent.com/40611140/112674872-11024a00-8e2c-11eb-80b4-49f661d08ab0.mov
